<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAnalysisWindow</class>
 <widget class="QMainWindow" name="DataAnalysisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据分析</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="selectLayout">
      <item>
       <widget class="QLabel" name="deviceLabel">
        <property name="text">
         <string>设备：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deviceComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="startDateTimeEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>至</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="endDateTimeEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="analysisButton">
        <property name="text">
         <string>分析</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>导出结果</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="chartWidget" native="true"/>
    </item>
    <item>
     <widget class="QTableWidget" name="resultTable"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 